ff/src/fields/macros.rs

```diff
@@ -717,5 +717,19 @@ macro_rules! impl_Fp {
                 self.0.zeroize();
             }
         }
+
+        impl<P: $FpParameters> From<num_bigint::BigUint> for $Fp<P> {
+            #[inline]
+            fn from(val: num_bigint::BigUint) -> $Fp<P> {
+                $Fp::<P>::from_repr(<$BigIntegerType>::from(val)).unwrap()
```

Comment:
For this, let's use `from_bytes_mod_order`, or convert `P::MODULUS` to `BigUint` and then use the modulo operator there.

Reply:
Done.